README.md

```diff
@@ -80,11 +80,11 @@ should return the ingredient instance
 - `RecipeIngredient#recipe`
 should return the recipe instance
 
-### `Allergen`
-An Allergen is a join between a user and an ingredient.  This is a has-many-through relationship.  What methods should an instance of this model respond to?
+### `Allergy`
```

Comment:
This subtlety below comes AFTER it's domain model impact is needed in `declare_allergy`. Move this bloc before `User`?